nativeApp/bin/touch_tsx/index.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Command line utility for a creating a blank template for components
+ *
+ * Usage:
+ *  npm run touch:tsx [PATH] [FILE (optional)]
+ * 
+ * If no file is supplied:
+ * - directory is created (if does not exist)
+ * - index file is created inside directory
+ * 
+ * If file is supplied
+ * - file is created inside of directory
+ *
+ * args:
+ *  PATH: path from root where the folder should be created
+ *  eg. ./src/open_views/screens/ForgotPassword
+ * 
+ * FILE: capitalised component name
+ *
+ */
+import getTemplate from './template';
+import { writeFileSync, existsSync, mkdirSync } from 'fs';
+
+const capitaliseFirst = (string) => string.charAt(0).toUpperCase() + string.substring(1);
+
+(() => {
```

Comment:
Don't think the IIFE is necessary here, the only reason I use it in scripts is because there's no top-level `await` in JS yet.